packages/hub/README.md

```diff
@@ -136,7 +136,10 @@ Documentation on how to create migration scripts is available at https://salsita
 After you have completed running your new DB migration script, update Prisma’s schema and client:
 
     yarn run prisma db pull
+    node dist/hub.js db transform-prisma-schema
+    yarn run prisma prisma format
     yarn run prisma generate
```

Comment:
Since skipping any one of these will result in invalid code or types, it feels like they should all be bundled into one command

Reply:
ya will do that in 4253 🤓